exec/java-exec/src/main/java/org/apache/drill/exec/expr/stat/RangeExprEvaluator.java

```diff
@@ -258,21 +258,28 @@ public long getRowCount() {
       final ValueHolder minFuncHolder = InterpreterEvaluator.evaluateFunction(interpreter, args1, holderExpr.getName());
       final ValueHolder maxFuncHolder = InterpreterEvaluator.evaluateFunction(interpreter, args2, holderExpr.getName());
 
+      Statistics<T> statistics;
       switch (destType) {
-      //TODO : need handle # of nulls.
       case INT:
-        return getStatistics( ((IntHolder)minFuncHolder).value, ((IntHolder)maxFuncHolder).value);
+        statistics = getStatistics( ((IntHolder)minFuncHolder).value, ((IntHolder)maxFuncHolder).value);
```

Comment:
`getStatistics( ((IntHolder)` remove unneede space, here and below.

Reply:
Done. Checked other occurrences.